sdk/react/src/lib/markers.js

```diff
@@ -21,5 +21,28 @@ export const parseInputArray = (markers: ?Array<string>): Array<Marker> => {
         return null;
       }
     })
-    .filter(marker => Boolean(marker));
+    .filter(marker => Boolean(marker))
+    .sort(({ start: a }, { start: b }) => {
```

Comment:
maybe add a test?

Reply:
@ooyala-amohnatkin okay, will do, please request changes for visibility